### PR TITLE
Jrenaat jrenaat wip/6.0 jpatest migrations 5

### PR DESCRIPTION
**Pull request by dreab8**

### CODE REVIEW

File: hibernate-core/src/test/java/org/hibernate/orm/test/jpa/transaction/batch/JtaWithStatementsBatchTest.java [hunk 1]
```diff
@@ -92,16 +92,18 @@ public void testUnableToReleaseStatementMessageIsNotLogged(EntityManagerFactoryS
 						assertStatementsListIsCleared();
 						assertAllStatementsAreClosed( testBatch.createdStatements );
 					}
-					catch (Exception e) {
+					catch (Throwable t) {
```
Comment (jrenaat) [inline hibernate-core/src/test/java/org/hibernate/orm/test/jpa/transaction/batch/JtaWithStatementsBatchTest.java:95]: I wonder if it wouldn't be better to catch both Exception + AssertionError, as you did in TransactionUtil.wrapInTransaction?

**Review by jrenaat — APPROVED** [review]
LGTM

File: hibernate-core/src/test/java/org/hibernate/orm/test/jpa/transaction/batch/JtaWithStatementsBatchTest.java (hunk 1 above)
Comment (dreab8) [reply hibernate-core/src/test/java/org/hibernate/orm/test/jpa/transaction/batch/JtaWithStatementsBatchTest.java:95]: changed it, thanks @jrenaat

**Comment by dreab8** [comment]
merged

**Pull request closed without merging**